api/src/db.js

```diff
@@ -12,6 +14,7 @@ const sequelize = new Sequelize(
     native: false, // lets Sequelize know we can use pg-native for ~30% more speed
   },
 );
+menuModel(sequelize)
```

Comment:
ponelo abajo del comentario DEFINE MODELS

Reply:
i think everything is ready to merge to feat/api , i did the pull of the feat/api , and solve conflict